tests/NCronJob.Tests/Helpers/EventsHelper.cs

```diff
@@ -0,0 +1,82 @@
+using Shouldly;
+
+namespace NCronJob.Tests;
+
+public static class EventsHelper
+{
+    public static IList<ExecutionProgress> WithOrchestrationId(
```

Comment:
Hmm not sure if the name sits right with me.
It might imply we Concat the events list with a new orch-Id.
Maybe?
 * FilterByOrchestrationId
 * ForOrchestration